recipes/nanopb/all/conandata.yml

```diff
@@ -0,0 +1,6 @@
+sources:
+  # Newer versions at the top
+  "0.4.8":
+    url:
+    - "https://github.com/nanopb/nanopb/archive/refs/tags/0.4.7.tar.gz"
+    sha256: "e1d89b166367a3c42d42311bdc2ed57ff78f86aede9a66828516e45490d48c81"
```

Comment:
You put `0.4.8` as the version, but the url points to version `0.4.7`

Reply:
```suggestion
  "0.4.8":
    url: "https://github.com/nanopb/nanopb/archive/refs/tags/0.4.8.tar.gz"
    sha256: "e1d89b166367a3c42d42311bdc2ed57ff78f86aede9a66828516e45490d48c81"
```